ui/plugins/pluginpanel.html

```diff
@@ -206,6 +206,13 @@
           <label>Log parsing status:</label>
           <span>No logs</span>
         </li>
+        <li class="small" ng-click="backfillJob()">
+          <label>
+            <a title="Trigger this jobtype back to the previous time it was run">
+                Backfill this job
+            </a>
+          </label>
+        </li>
```

Comment:
The Job details panel is for passive actions (viewing, reading). If this is a flavor of retrigger imo I think it's more appropriate in the original location you had it (Job details action bar).